build/template-nuget-pack.yaml

```diff
@@ -0,0 +1,13 @@
+
+parameters:
+  BuildConfiguration: 'release'
+  ProjectPath: ''
+
+steps:
+- task: VSBuild@1
+  displayName: 'Pack ${{ parameters.ProjectPath }}'
+  inputs:
+    solution: ${{ parameters.ProjectPath }}
+    vsVersion: '15.0'
+    msbuildArgs: '/t:pack /p:nobuild=true /p:IncludeSymbols=true'
```

Comment:
Note that I had found and logged [a bug with the msbuild.sdk.extras](https://github.com/onovotny/MSBuildSdkExtras/issues/136) where /t:pack would actually rebuild the project even if /p:nobuild=true is passed. The workaround was to pack twice (see details in the bug). 

So just a heads up to check that the assemblies in the nuget package are actually signed. If they are not - it's probably because of this.

Reply:
I've validated that the binaries are signed.  What's running here is exactly what we have running in our existing builds, just broken up into yaml scripts.